### ISSUE TITLE
Auto create device in production app when switch apps from lab to production

### ISSUE THREAD

**Issue by cmydlarz** [floodsense/floodnet-nodered#1]
When moving a device between apps the confirm button should auto create a new device in the production/deployed app prior to the sending of the downlink that actually performs the appkey change

Question for @rictolcrow and @Praneethsvch : should we delete the device from the lab app? Maybe to keep things clean?

**Comment by rictolcrow** [comment]
If we delete the device from the lab app, can we reverse the process? i.e. move from deployed app to lab app

**Comment by cmydlarz** [comment]
> If we delete the device from the lab app, can we reverse the process? i.e. move from deployed app to lab app

Yes, using the same process but just setting the lab app as the target. The ttn/tti and helium apis allow for device creation and deletion so I think it can all be automated

**Comment by rictolcrow** [comment]
I can ping-pong back and forth between apps. Nice! End device creation doesn't work if DeviceID, AppEUI and DevEUI are the same in both applications. I changed AppEUI from CNYAPPXX to CNYDEPXX and autogenerated AppKey.

**Comment by Praneethsvch** [comment]
> When moving a device between apps the confirm button should auto create a new device in the production/deployed app prior to the sending of the downlink that actually performs the appkey change
> 
> Question for @rictolcrow and @Praneethsvch : should we delete the device from the lab app? Maybe to keep things clean?

The test app could be useful as a place to fall back for the end-device when app-change fails. Further, app-change can be retried from test-app again
